### PR TITLE
add click rule for arte.tv

### PR DESCRIPTION
**Pull request by arxo**
for issue  #211

### CODE REVIEW

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -1,6 +1,15 @@
 {
   "$schema": "./CookieBannerRuleList.schema.json",
   "data": [
+    {
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:4]: Please add the rule at the bottom of the file.

**Review by Trikolon — CHANGES_REQUESTED** [review]
Thanks! The rule works well, just some smaller issues we should address before merging. See comments inline.

File: cookie-banner-rules-list.json [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+      "id": "arte.tv_via_trustcommander",
```
[the remark this answers is not in the document]
Comment (arxo) [reply cookie-banner-rules-list.json]: updated accordingly

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (arxo) [reply cookie-banner-rules-list.json]: updated accordingly

**Comment by Trikolon** [comment]
Prettier isn't happy with the latest change. Try running `npm run prettier`.

**Comment by arxo** [comment]
updated formatting, should be good now

**Review by Trikolon — APPROVED** [review]
Awesome, thanks!

**Pull request merged**